CHANGES.txt

```diff
@@ -3,9 +3,11 @@ Comtypes CHANGELOG
 
 Release 1.1.11
 -------------
- * Fix Python 3 compatibility. Thanks @kdschlosser.
- * Fix `tlib_mtime` in the absence of typelib.
+ * Fix setuptools>=60.0.0 compatibility. Thanks @kdschlosser.
```

Comment:
I think the version that introduced the [problem](https://github.com/robocorp/comtypes-fork/pull/1/files#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7R21) was actually `57.5.0`.